rules/REQUEST-944-APPLICATION-ATTACK-JAVA.conf

```diff
@@ -142,6 +142,45 @@ SecRule ARGS|ARGS_NAMES|REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES
     setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
 
+#
+# [ Java Script Uploads ]
+#
+# Block file uploads with filenames ending in Java scripts (.jsp, .jspx)
+#
+# Many application contain Unrestricted File Upload vulnerabilities.
+# https://www.owasp.org/index.php/Unrestricted_File_Upload
```

Comment:
Maybe updating link to new url: https://owasp.org/www-community/vulnerabilities/Unrestricted_File_Upload ?

Reply:
Updated.